src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -31,7 +31,8 @@ class PressableWithSecondaryInteraction extends Component {
      * @param {Object} e
      */
     callSecondaryInteractionWithMappedEvent(e) {
-        if (e.nativeEvent.state !== State.ACTIVE) {
+        const isHoverableDevice = ((Platform.OS === 'web') || (Platform.OS === 'windows') || (Platform.OS === 'macos'));
+        if ((e.nativeEvent.state !== State.ACTIVE) || isHoverableDevice) {
```

Comment:
Doing platform-specific code this way is not allowed. If you want to run some code on a platform use files like `index.web.js or index.android.js or index.native.js` etc. there are a few libs in src/libs folder which you can take as an example.

Reply:
Also, Could you please explain to me what are you trying to do here? I can offer alternatives.

Reply:
Thanks for the detailed response. My purpose here is not to create platform-based code, but to check whether the device is a hoverable device or not. On Hoverable device the Long press handler perceives it as holding it while scrolling and puts us in the action function. I disable the long press handler on hoverable devices with this control.

Reply:
At the same time, if I write this file separately for each platform like `index.web.js or index.android.js or index.native.js` the same file will be repeated for some platforms. Therefore there will be more than one of the same file.

Reply:
Ok, so you mean that you want to do something for Web and Desktop.  So create like this
```
isHoverable
 index.native.js returns false;
 index.js  returns true
 ```

Reply:
Yes it seems like a good idea, thanks. Then I convert it to like this and test it. After the tests, I will update the pr and let you know.

Reply:
A better name would be hasHoverSupport

Reply:
Changed the name and made 1 more commit 👍🏼